Project.ConsoleApp.Test/ConsoleAppTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+
+namespace Project.ConsoleApp.Test
+{
+    [TestClass]
+    public class ConsoleAppTest
+    {
+        string _validJSONFile = string.Empty;
+        string _invalidJSONFile = string.Empty;
+        string _validXMLFile = string.Empty;
+        string _invalidXMLFile = string.Empty;
+
+        public ConsoleAppTest()
+        {
+            string testFilePath = Directory.GetCurrentDirectory() + @"\TestFiles\";
+
+            _validJSONFile = testFilePath + "JSON_ValidFormat.json";
+            _invalidJSONFile = testFilePath + "JSON_InvalidFormat.json";
+            _validXMLFile = testFilePath + "XML_ValidFormat.xml";
+            _invalidXMLFile = testFilePath + "XML_InvalidFormat.xml";
+        }
+
+        [TestMethod]
+        public void AllTestFilesExist()
+        {
+            Assert.IsTrue(File.Exists(_validJSONFile));
```

Comment:
`Purposeful Code`: This test doesn't test any functionality.  It only tests that you set up your test project correctly.  If a file is deleted (presumably by mistake) this your tests will fail.  This test is very similar to `int i = 3; Assert.IsTrue(i == 3);`